integration-fe/src/sagas/getRooms.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-debugger */
+import { call, put } from 'redux-saga/effects';
+import { getRooms as getUserRooms } from './apiCalls';
```

Comment:
This saga executes an async operation in order to fetch the user rooms. After fetch, this dispatch an action to update the application state.